src/artboard.cpp

```diff
@@ -375,7 +375,7 @@ bool Artboard::updateComponents() {
 
 bool Artboard::advance(double elapsedSeconds) {
     for (auto nestedArtboard : m_NestedArtboards) {
-        nestedArtboard->advance(elapsedSeconds);
+        nestedArtboard->advance((float)elapsedSeconds);
```

Comment:
I suppose this will quit being precise to 1/60th of a second and start janking after 2^24/60 seconds ~= 200 days. Do we care? Can we fmod or clamp it before truncating?

Reply:
Actually, is this “elapsed since last time you called advance”, or “elapsed since the animation began”?

Reply:
Elapsed since last call!

Reply:
I agree that we might care -- we might make all the delta-durations doubles in the future.